core/rend/dx11/dx11_renderer.cpp

```diff
@@ -185,6 +185,12 @@ void DX11Renderer::Term()
 	quad.reset();
 	deviceContext.reset();
 	device.reset();
+#ifdef VIDEO_ROUTING
+	vrStagingTexture.reset();
+	vrStagingTextureSRV.reset();
+	vrScaledTexture.reset();
+	vrScaledRenderTarget.reset();
+#endif
```

Comment:
no need for #ifdef here. It's not a problem if these objects are defined and reset in all builds.